wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/SuccessfulCertificateEnrollmentViewController.swift

```diff
@@ -145,26 +143,24 @@ final class SuccessfulCertificateEnrollmentViewController: AuthenticationStepVie
         NSLayoutConstraint.activate([
             // title Label
             titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 32),
-            titleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -32),
+            view.trailingAnchor.constraint(equalTo: titleLabel.trailingAnchor, constant: 32),
 
             // shield image view
             shieldImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            shieldImageView.heightAnchor.constraint(equalToConstant: 64),
-            shieldImageView.widthAnchor.constraint(equalToConstant: 64),
 
             // confirmation button
             confirmationButton.heightAnchor.constraint(equalToConstant: 56),
 
             // stackView
             stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24),
-            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -24),
+            view.trailingAnchor.constraint(equalTo: stackView.trailingAnchor, constant: 24),
             stackView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
 
             // certificate details button
             certificateDetailsButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 24),
-            certificateDetailsButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -24),
+            view.trailingAnchor.constraint(equalTo: certificateDetailsButton.trailingAnchor, constant: 24),
             certificateDetailsButton.heightAnchor.constraint(equalToConstant: 32),
-            certificateDetailsButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -64)
+            view.bottomAnchor.constraint(equalTo: certificateDetailsButton.bottomAnchor, constant: 64)
```

Comment:
question: Outside of removing the heightAnchor and widthAnchor which I do understand. Why the rest of the changes are needed? The reason I'm asking is because I don't see a screenshot in the PR with the new changes which makes it harder to review.

Reply:
Not needed, I'll revert it.